pages/Advanced Queries.md

```diff
@@ -39,22 +39,87 @@ description:: Advanced queries are written with [Datalog](https://en.wikipedia.o
+					- `:+1m-23` will result in the millisecond timestamp one month from today, at 11:00:00.000 pm
+				- `-HHMM` - Specific hour and minute of the day you are querying, in 24 hour time.  *(examples within)*
+				  collapsed:: true
+					- `:+1d-1430` will result in the millisecond timestamp for tomorrow, at 2:30:00.000 pm
+					- `:-2w-0000` will result in the millisecond timestamp 14 days ago, at 00:00:00.000 am
+					- `:+1m-2359` will result in the millisecond timestamp one month from today, at 11:59:00.000 pm
+				- `-HHMMSS` - Specific hour, minute, and seconds of the day you are querying for, in 24 hour time.  *(examples within)*
+				  collapsed:: true
+					- `:+1d-143015` will result in the millisecond timestamp for tomorrow, at 2:30:15.000 pm
+					- `:-2w-000000` will result in the millisecond timestamp 14 days ago, at 00:00:00.000 am
+					- `:+1m-235959` will result in the millisecond timestamp one month from today, at 11:59:59.000 pm
+				- `-HHMMSSmmm` - Specific hour, minute, seconds, and milliseconds of the day you are querying for, in 24 hour time.  *(examples within)*
+				  collapsed:: true
+					- `:+1d-143015777` will result in the millisecond timestamp for tomorrow, at 2:30:15.777 pm
+					- `:-2w-000000000` will result in the millisecond timestamp 14 days ago, at 00:00:00.000 am
+					- `:+1m-235959999` will result in the millisecond timestamp one month from today, at 11:59:59.999 pm
+				- `-ms` - when querying for dates in the past, it will return the timestamp 00:00:00.000, and when querying for dates in the future, will return the timestamp for 23:59:59.999.  *(examples within)*
+				  collapsed:: true
+					- `:-1d-ms` will return the timestamp for midnight the night before yesterday, at 00:00:00.000.
+						- Functionally equivalent to `:-1d-start`
+					- `:+1d-ms` will return the timestamp just before midnight tomorrow night, at 23:59:59.999.
+						- Functionallty equivalent to `:+1d-end`
+					-
+			- It should be noted that most **time suffixes** (all besides `-ms`) can be used in combination with the `:today` special date. *(examples within)*
+			  collapsed:: true
+				- `:today-start` - returns a timestamp for 00:00:00.000 today
+				- `:today-end` - returns a timestamp for 23:59:59.000 today
+				- `:today-HH` - returns a timestamp for HH:00:00.000 today
+				- `:today-HHMM` - returns a timestamp for HH:MM:00.000 today
+				- `:today-HHMMSS` - returns a timestamp for HH:MM:SS.000 today
+				- `:today-HHMMSSmmm` - returns a timestamp for HH:MM:SS.mmm today
+		- #### Deprecated inputs
+		  collapsed:: true
+			- Previously we supported different inputs between basic queries and advanced queries. These have now been unified, however we still support the follow queries for the time being. If you are using any of the following, plus upgrade them to the newer syntax mentioned for each item
+			- `:Xd` - Date for X number of days after today. **New syntax:** `:+Xd`
```

Comment:
Confusingly enough this used to match on -7d and +7d but it always short circuited on -7d. Glad we have the new syntax. I can fix this and related examples below